README.md

```diff
@@ -33,14 +33,22 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. What is the difference between an adaptive website and a fully responsive website?
+Responsive websites change and adapt regardless of the screen size being used while Adaptive design uses fixed sizes based on pre-determined breakpoints to deliver the most appropriate layout version for the screen size that is detected when the page first loads. Adaptive is easier to use but Responsive allows for adaptation and it supports all screen sizes as opposed to only the pre-set breakpoints determined in an adaptive approach.
 
 2. Describe what it means to be mobile first vs desktop first.
+Mobile first- begins with designing for the main function of the site, then aims for higher resolutions.
+
+Desktop first- means designing for the highest specifications — the best achievable by design- and then going down to smaller resolutions.
 
 3. What does `font-size: 62.5%` in the `html` tag do for us when using `rem` units?
+Root elements are relative to font-size. A font size of 62.5% is a rem of 6.25.
```

Comment:
A `rem` by default is equal to about 16 px in height. We as developers set the `font-size` of the `html` tag at 62.5 to make calculations when deciding on how large we want our text, we can calculate `1 rem` to equal 10px instead of 16, since base 10 is easier to work with than calculating by the default value.